src/test/ui/treat-err-as-bug/err.rs

```diff
@@ -0,0 +1,11 @@
+// compile-flags: -Ztreat-err-as-bug
+// failure-status: 101
+// rustc-env:RUST_BACKTRACE=1
+// error-pattern: aborting due to `-Z treat-err-as-bug=1`
+// error-pattern: rustc_mir::const_eval::eval_queries::const_eval_raw_provider
+// normalize-stderr-test "\s+ at .*" -> ""
```

Comment:
Can you also try to normalize away all backtrace lines but the `rustc_mir::const_eval::eval_queries::const_eval_raw_provider::{{closure}}`? Not sure if regex has something to match everything *but* a full word